lib/ui/addon/components/toolbar-divider.js

```diff
@@ -0,0 +1,7 @@
+import Component from '@ember/component';
```

Comment:
Could we make this a template-only-glimmer-component and remove this JS file?

Reply:
I'm not sure we can do that until 3.14. 🤷‍♀

Reply:
We can, it has been a feature of https://github.com/emberjs/ember-optional-features for awhile. I think we need to run `ember feature:enable template-only-glimmer-components` then delete the JS file.

Reply:
The plan in my head was to made a pass at components like I did in this PR, then go over again and _Glimmerize_ in another PR.

Reply:
I would just handle this one instance now, so we can get rid of the JS file, which we are only keeping around for `tagName: ''`

Reply:
Done and done.